BridgeSDK/BridgeAPI/SBBUploadManager.h

```diff
@@ -51,6 +51,18 @@ typedef void (^SBBUploadManagerCompletionBlock)(NSError *error);
  */
 - (void)uploadManager:(SBBUploadManager *)manager uploadOfFile:(NSString *)file completedWithError:(NSError *)error;
 
+@optional
+/*!
+ Optional delegate method available only in debug mode.
+ Called when a background file upload task finishes, successfully or otherwise in debug mode.
+ Provides a URL that can be called for more information about the post-upload status of the data.
+ 
+ @param manager The upload manager instance making the call.
+ @param file    The path to the file whose upload task finished or failed.
+ @param url     The url to call to get more information about the post-upload status of the data.
+ */
+- (void)uploadManager:(SBBUploadManager *)manager uploadOfFile:(NSString *)file completedWithVerificationURL:(NSURL *)url;
+
```

Comment:
Instead of "debug mode" (in both places it's used) I'd say "Debug builds" since it's a compile-time thing, not a runtime mode setting.

I had also been thinking this declaration and the accompanying headerdoc comments should be wrapped in the conditional compilation macros but given that it's optional, and that there's no way to enforce the implementation of the method also being wrapped in the conditional macros, I guess there's no point. So never mind on that.